srcpkgs/wlroots/template

```diff
@@ -12,6 +12,7 @@ makedepends="elogind-devel libcap-devel wayland-devel wayland-protocols
  libxcb-devel xcb-util-cursor-devel xcb-util-devel xcb-util-image-devel
  xcb-util-keysyms-devel xcb-util-renderutil-devel xcb-util-wm-devel
  xcb-util-errors-devel xcb-util-xrm-devel"
+depends="elogind"
```

Comment:
What's happening if elogind is missing?

Reply:
The first time I launched sway, my PC displaied this and just freezed/crashed:
`[backend/session/logind.c:489] Failed to get session id: No data available`
I installed elogind and the problem was over.

I've tried to remove elogind package (after rebuilding sway with less dependencies) and now it seems to work correctly.